multiqc/modules/genomescope2/genomescope2.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from genomescope """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re, base64
+from multiqc.plots import table
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """Genomescope2 module"""
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name="Genomescope2",
+            anchor="Genomescope2",
+            href="https://github.com/tbenavi1/genomescope2.0",
+            info="Reference-free profiling of polyploid genomes.",
+            doi="https://doi.org/10.1038/s41467-020-14998-3",
```

Comment:
DOIs should be short-form:

```suggestion
            doi="10.1038/s41467-020-14998-3",
```